mmv1/openapi_generate/parser.go

```diff
@@ -451,7 +451,12 @@ func buildProperties(props openapi3.Schemas, required []string) []*api.Type {
 
 // Trims whitespace from the ends of lines in a description to force multiline
 // formatting for strings with newlines present
-func trimSpacesFromDescription(description string) string {
+// Also trim "Output only." and "Required." from descriptions as this gets duplicated
+func trimDescription(description string) string {
```

Comment:
This should potentially do a loop while a prefix remains to address cases like https://github.com/GoogleCloudPlatform/magic-modules/blob/8ce3b475fd363aea309851730f35bd82d97044a9/mmv1/products/developerconnect/GitRepositoryLink.yaml#L96

Reply:
ignoring this as a one-time issue